### PR TITLE
Customize Preferences for my Child

### PR DESCRIPTION
**Pull request by wSedlacek**
# Description

As a Parent, I would like to set my child's accessibility settings, so that my child can participate without struggling

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# Checklist

- [ ] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by wSedlacek** [comment]
[[Parent] - Customize Preferences for my Child](https://trello.com/c/nVnDmDO9/35-parent-customize-preferences-for-my-child)

**Review by ghost — COMMENTED** [review]
sort children list.

File: src/routes/canon/canon.routes.ts [hunk 1]
```diff
@@ -25,9 +25,18 @@ canonRoutes.get('/:id', async (req, res) => {
...
-        const canon = await getRepository(Canon, connection()).findOne(req.params.id);
-        if (!canon) throw new Error('404');
-        res.json({ canon });
+        const cannonIndex = await getRepository(Canon, connection()).findOne(req.params.id);
+        if (!cannonIndex) throw new Error('404');
+
+        if (req.user instanceof Child) {
+            res.json({
+                canon: req.user?.preferences.dyslexia
+                    ? { base64: cannonIndex.altbase64 }
```
Comment (EricBanker12) [inline src/routes/canon/canon.routes.ts]: altbase64 is nullable, in which case the front end crashes. 

changing `? { base64: cannonIndex.altbase64 }` to `? { base64: cannonIndex.altbase64 || cannonIndex.base64}` is one way to fix it

**Review by EricBanker12 — COMMENTED** [review]
One change is needed for the edge case where the admin has not set a dyslexic friendly version. Otherwise all tests and manual testing appear to be working correctly.

**Review by EricBanker12 — APPROVED** [review]
The issue with null returns has been fixed, and all tests still pass.
